common/OpTestHMC.py

```diff
@@ -305,7 +306,11 @@ def wait_login_prompt(self, console, username=None, password=None):
                     elif i == 1:
                         raise OpTestError("Wrong Credentials for host")
                     else:
-                        raise OpTestError("Upexpected return")
+                        last_count = last_count+1
+                        if last_count == 2:
+                            raise OpTestError("Upexpected return")
+                        self.wait_login_prompt(console, self.lpar_user, self.lpar_password)
```

Comment:
Should this be a loop instead of a recursive call?

Reply:
@sammj sure make sense i added while loop logic instead of recursive call